appserver/admingui/war/src/main/webapp/WEB-INF/sun-web.xml

```diff
@@ -61,6 +61,6 @@
 	<parameter-encoding default-charset="UTF-8" />
  </locale-charset-info>
 
-<class-loader delegate="true" extra-class-path="WEB-INF/extra/webui-jsf-suntheme-4.0.2.10.payara-p15.jar:WEB-INF/extra/dojo-ajax-nodemo-0.4.4.jar:WEB-INF/extra/webui-jsf-4.0.2.10.payara-p15.jar:WEB-INF/extra/commons-fileupload-1.3.3.jar:WEB-INF/extra/commons-io-2.6.jar:WEB-INF/extra/json-1.0.jar:WEB-INF/extra/prototype-1.5.0.jar" />
+<class-loader delegate="true" extra-class-path="WEB-INF/extra/webui-jsf-suntheme-4.0.2.10.payara-p15.jar:WEB-INF/extra/dojo-ajax-nodemo-0.4.4.jar:WEB-INF/extra/webui-jsf-4.0.2.10.payara-p15.jar:WEB-INF/extra/commons-fileupload-1.3.3.jar:WEB-INF/extra/commons-io-2.6.jar:WEB-INF/extra/json-1.0.jar:WEB-INF/extra/prototype-1.7.3.jar" />
```

Comment:
Has the suntheme not also been upgraded to p16?

Reply:
No, turns out that a woodstock update isn't required. I've reverted the patched projects commit as well.